tests/pytests/functional/formulas/test_docker.py

```diff
@@ -0,0 +1,21 @@
+"""
+Tests using docker formula
+"""
+import pytest
+
+
+@pytest.fixture(scope="module")
+def _formula(saltstack_formula):
+    with saltstack_formula(name="docker-formula", tag="2.4.2") as formula:
+        yield formula
+
+
+@pytest.fixture(scope="module")
+def modules(loaders, _formula):
+    return loaders.modules
+
+
+def test_docker_formula(modules):
+    ret = modules.state.sls("docker", test=True)
```

Comment:
Why are we running all of these formulas with `test=True`?

We want the formula's to run to ensure the states run properly.

Reply:
Looks like Salt is not finding the downloaded formulas:
`MultiStateResult(raw=["No matching sls found for 'docker' in env 'base'"], _structured=[])`